Src/GoogleApis.Auth/OAuth2/ITokenAccess.cs

```diff
@@ -22,19 +22,24 @@
 namespace Google.Apis.Auth.OAuth2
 {
     /// <summary>
```

Comment:
Same for https://github.com/google/google-api-dotnet-client/blob/592fef8270cc83f0ef9477069d8da9ba21f40cf5/Src/GoogleApis.Auth.DotNet4/OAuth2/JsonCredentialParameters.cs

Sorry I didn't notice that before.